generator/uscore/templates/sequence.rb.erb

```diff
@@ -7,7 +7,7 @@ module Inferno
   module Sequence
     class <%=class_name%> < SequenceBase
       include Inferno::DataAbsentReasonChecker  
-      include Inferno::USCore310ProfileDefinitions
```

Comment:
I'm confused why you are making this version independent.  I'd think the profile definitions are version specific?

Reply:
The class is version dependent while module should not be. 
![image](https://user-images.githubusercontent.com/8761175/92618513-0f08dd00-f286-11ea-9601-4f852501d269.png)

Reply:
I should regenerate v3.1.0 tests too.

Reply:
What was the reason for this change? `USCore311CarePlanSequenceDefinitions` will never be instantiated, so it makes sense for it to be a module that gets included rather than a class living in a module that gets included.

Reply:
Oh ok.  Yes it is critical that this does not break US Core v3.1.0 tests if you regenerate them (or not).  And we need to get whatever you've changed in here back to Program so we don't fall out of synch.

Reply:
If making the generator generic between the two is too big a lift for today, we could merge in just the tests to this release and leave your generator changes in a separate branch to be completed later.

Reply:
@arscan Yes. Remove version from module name is critical to allow different US Core version lives side by side. 
@Jammjammjamm I am trying to minimize changes while adopting 3.1.1 so the Profile Definition module is based on existing 3.1.0 branch. I can create a JIRA ticket to optimize this area.

Reply:
Created https://oncprojectracking.healthit.gov/support/browse/FI-1004 for generating SequenceDefinition as a module.

Reply:
We fixed this to keep the same naming convention.